src/force-directed.ts

```diff
@@ -142,22 +232,70 @@ const vis: ForceDirectedGraphVisualization = {
       }
     };
 
-    // If color_picker is undefined, chances are
-    // all other config attributes are also undefined
-    if (config.color_range === undefined) {
-      applyDefualtConfig();
+    for (let opt in this.options) {
```

Comment:
Some lingering dashboard-next problems during development. I think you're right though -- once the config is registered I think we're good to go. We'll have to keep an eye on the new triggers on db-next still since I'm still getting some odd behavior 🤷